toolset/setup/linux/scaffolding.py

```diff
@@ -0,0 +1,236 @@
+import sys
+import os
+import subprocess
+from shutil import copytree
+
+class Scaffolding:
+  def __init__(self):
+    print """
+-------------------------------------------------------------------------------
+    This wizard is intended to help build the scaffolding required for a new 
+    test to be benchmarked.
+
+    From here, you will be prompted for values related to the test you
+    wish to add.
+-------------------------------------------------------------------------------"""
+
+    # try:
+    self.__gather_display_name()
+    self.__gather_language()
+    self.__gather_approach()
+    self.__gather_classification()
+    self.__gather_orm()
+    self.__gather_webserver()
+    self.__gather_versus()
+
+    self.__confirm_values()
+    # except:
+      # print ""
+
+  def __gather_display_name(self):
+    print """
```

Comment:
One of the things I've been doing is making sure that any new python code is compatible with Python 3 for an easier path to upgrade the toolset. One of the breaking changes is that `print` is no longer a statement in Python 3, but a function. https://docs.python.org/3/whatsnew/3.0.html

So I've been trying to keep that consistent https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/toolset/run-tests.py#L74

Reply:
Got it.